src/All.php

```diff
@@ -0,0 +1,155 @@
+<?php namespace Leean\Acf;
+
+/**
+ * Class to provide helpers for getting a all ACF fields for an entity.
+ */
+class All
```

Comment:
The classes are really just groups of methods. So all the methods in the All class retrieve all fields, and the ones in the Single class for single.
However, I just thought we could have one class called Acf and overload the functions (the $field_name would be optional). I think this is better, right?